BadgeUpClient/ResourceClients/CriterionClient.cs

```diff
@@ -15,13 +19,39 @@ public CriterionClient(BadgeUpHttpClient httpClient)
 		}
 
 		/// <summary>
-		/// Retrieves a criterion by ID
+		/// Retrieves a criterion by ID.
 		/// </summary>
 		/// <param name="id">A string that uniquely identifies this criterion</param>
 		/// <returns><see cref="CriterionResponse"/></returns>
 		public async Task<CriterionResponse> GetById(string id)
 		{
 			return await this.m_httpClient.Get<CriterionResponse>(ENDPOINT + "/" + id);
 		}
+
+		/// <summary>
+		/// Retrieves a list of all criteria.
+		/// </summary>
+		/// <returns>The list of all criteria.</returns>
+		public async Task<List<CriterionResponse>> GetAll()
+		{
+			return await this.m_httpClient.GetAll<CriterionResponse>(ENDPOINT);
+		}
+
+		/// <summary>
+		/// Creates a single criterion with the given parameters.
+		/// </summary>
+		/// <param name="criterion">The criterion to create.</param>
+		/// <returns>The created criterion.</returns>
+		public async Task<CriterionResponse> CreateCriterion(Criterion criterion)
```

Comment:
Would we not just want to call these `Create` since they are a part of the `CriterionClient`?
If the answer is no, then perhaps we should rename `GetAll` to `GetAllCriteria`

Reply:
I agree, `.Create()` is generally better, for both Achievement and Criterion. I'll fix it. 
I think I followed convention from somewhere else in the codebase, but I can't find the origin at the moment, so it's probably an oversight.